### ISSUE TITLE
[Bug] regenerator-runtime not found

### ISSUE THREAD

**Issue by lorensr** [temporalio/sdk-typescript#534]
Version: `0.19.0`

Running a worker with `ts-node` sometimes* results in `swc-loader` including code with `require('regenerator-runtime')` in the bundle, which Webpack tries to resolve, and fails.

Related issue in `swc`: https://github.com/swc-project/swc/issues/1461

*sometimes: works in `samples-typescript` repo, but not when using `@temporalio/create`

### works

```
git clone https://github.com/temporalio/samples-typescript.git
cd samples-typescript/hello-world
npm i temporalio@0.19.0
npm start
```

### errors

```
npx @temporalio/create@latest ./example
cd example
npm i temporalio@0.19.0 @temporalio/worker@0.19.0
npm start

...
2022-03-12T23:27:08.424Z [ERROR] ERROR in ./src/workflows.ts 30:0-53
2022-03-12T23:27:08.424Z [ERROR] Module not found: Error: Can't resolve 'regenerator-runtime' in '/Users/me/test/example/src'
```

In both locations, `tsc && node lib/worker` works.

**Comment by julianocomg** [comment]
@lorensr  Did you try installing regenerator-runtime as a dependency?

**Comment by lorensr** [comment]
Yes, adding it as a devDep is a workaround, but we'd like to figure out when/why it's included in the webpack bundle in the first place (we'd like to keep bundle size down, and it shouldn't be needed for our build target, Node 14+).

**Comment by lorensr** [comment]
> We'll definitely want to support swc-loader we will consider making it the default, I just want to verify it works as expected and doesn't transform generators and async functions.
—Roey

**Comment by julianocomg** [comment]
@lorensr can’t wait for that. thank you 🙏

**Comment by mjameswh** [comment]
If I paste the following code on https://swc.rs/playground, it seems that `regenerator-runtime` and `_asyncToGenerator` completely disappear once `target` is set at least to ES2017... Could be worth trying setting this option through `swc-loader`'s options...

```
import { y } from 'abc';

async function asyncFunc() {
    await y();
}

function *genFunc() {
    yield 'a';
    yield 'b';
    yield 'c';
}
```

**Comment by lorensr** [comment]
`swc-loader` included in `0.20.2`
